extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/supervisor/KafkaSupervisorTest.java

```diff
@@ -574,222 +573,10 @@ public void testBadMetadataOffsets() throws Exception
     supervisor.runInternal();
   }
 
-  @Test
-  public void testKillIncompatibleTasks() throws Exception
-  {
-    supervisor = getSupervisor(2, 1, true, "PT1H", null, null);
-    addSomeEvents(1);
-
-    // unexpected # of partitions (kill)
-    Task id1 = createKafkaIndexTask(
-        "id1",
-        DATASOURCE,
-        1,
-        new SeekableStreamPartitions<>("topic", ImmutableMap.of(0, 0L)),
-        new SeekableStreamPartitions<>("topic", ImmutableMap.of(0, 10L)),
-        null,
-        null
-    );
-
-    // correct number of partitions and ranges (don't kill)
-    Task id2 = createKafkaIndexTask(
-        "id2",
-        DATASOURCE,
-        0,
-        new SeekableStreamPartitions<>("topic", ImmutableMap.of(0, 0L, 1, 0L, 2, 0L)),
-        new SeekableStreamPartitions<>("topic", ImmutableMap.of(0, 333L, 1, 333L, 2, 333L)),
-        null,
-        null
-    );
-
-    // unexpected range on partition 2 (kill)
-    Task id3 = createKafkaIndexTask(
-        "id3",
-        DATASOURCE,
-        1,
-        new SeekableStreamPartitions<>("topic", ImmutableMap.of(0, 0L, 1, 0L, 2, 1L)),
-        new SeekableStreamPartitions<>("topic", ImmutableMap.of(0, 333L, 1, 333L, 2, 330L)),
-        null,
-        null
-    );
-
-    // different datasource (don't kill)
```

Comment:
Can you make sure you have equivalent cases in the replacement tests that you added?